schemas/context/experienceevent-media.example.1.json

```diff
@@ -123,7 +148,9 @@
       },
       "xdm:pauseTime": {
         "xdm:value": 0
-      }
+      },
+      "xdm:idp": "Hulu",
+      "xdm:mediaAuth": "FALSE"
```

Comment:
improper encoding of a boolean value

Reply:
This is not a boolean variable. If usually has a true/false value, but the clients can send any text value. Will add some extra examples with different values to make it more clear.